completions/ssh

```diff
@@ -108,15 +108,21 @@ _ssh_suboption()
         canonicalizehostname)
             COMPREPLY=( $(compgen -W 'yes no always' -- "$cur") )
             ;;
-        *file|identityagent|include)
+        *file|identityagent|include|controlpath|proxycommand|revokedhostkeys|xauthlocation)
```

Comment:
Wouldn't proxycommand be better completed with `compgen -c -- "$cur"` than `_filedir`?

Reply:
You are right. In this context, I noticed that I missed also the `LocalCommand` and `RemoteCommand` options, so I added them to the same branch as `ProxyCommand` in the last commit